ducktape/src/main/scala/io/github/arainko/ducktape/internal/modules/MaterializedConfiguration.scala

```diff
@@ -14,6 +14,7 @@ private[ducktape] object MaterializedConfiguration {
     case Const(destFieldName: String, value: Expr[Any])
     case Computed(destFieldName: String, fuction: Expr[Any => Any])
     case Renamed(destFieldName: String, sourceFieldName: String)
+    case Default(destFieldName: String)
```

Comment:
I think we should reuse `Const` for this use case since we know it's implemented properly (like I did with `Field.allMatching`) and push all of the logic of failing etc. to `materializeSingleProductConfig`